SU2_CFD/src/output/CAdjElasticityOutput.cpp

```diff
@@ -62,6 +62,11 @@ CAdjElasticityOutput::CAdjElasticityOutput(CConfig *config, unsigned short nDim)
     nRequestedVolumeFields = requestedVolumeFields.size();
   }
 
+  if (find(requestedVolumeFields.begin(), requestedVolumeFields.end(), string("SENSITIVITY")) == requestedVolumeFields.end()) {
```

Comment:
:+1: I think this way of what happens in the output stays in the output is clearer, even if it requires duplicating a bit of code across the different adjoint output classes.

Reply:
Alright, thanks for your feedback on this :)